apps/weshipit/pages/onboarding.tsx

```diff
@@ -1,15 +1,24 @@
 import React from 'react';
 import { linksApi } from './api/links';
 import Layout from '../components/layout';
+import type { Metadata } from 'next';
+
+export const metadata: Metadata = {
+  title: 'Onboard with React Native Experts',
+  description:
+    'We are excited to work with you! Please fill out the form below to get started.',
+};
 
 export default function Onboarding() {
   return (
     <Layout
       withContainer
       withFooter
-      seoTitle="Onboard with React Native Experts"
-      seoDescription="We are excited to work with you! Please fill out the form below to get started."
       withHeader
+      seoTitle={'Onboard with React Native Experts'}
```

Comment:
instead of duplicating the content you can use `metadata.title` and `metadata.description`.